lib/sengiri/model/base.rb

```diff
@@ -66,16 +66,32 @@ def sharding_group(name, confs: nil, suffix: nil)
 
         def dbconfs
           if defined? Rails
-            @dbconfs ||= Rails.application.config.database_configuration.select {|name|
-              /^#{@sharding_group_name}/ =~ name
-            }.select {|name|
-              /#{env}#{@sharding_database_suffix}$/ =~ name
-            }
-
+            @dbconfs ||= rails_db_confs
           end
           @dbconfs
```

Comment:
You mean `@dbconfts` raises error in method chains ?
i think it will be not.

```rb
Rails.application.config.database_configuration.select { false }.select { false }
=> {}
```

Reply:
@YumaInaura I said that `Sengiri::Model::Base#dbconf`  can raise error
Could you check this line? https://github.com/mewlist/sengiri/blob/master/lib/sengiri/model/base.rb#L66-L68